runtime/compiler/optimizer/J9RecognizedCallTransformer.cpp

```diff
@@ -367,6 +387,9 @@ bool J9::RecognizedCallTransformer::isInlineable(TR::TreeTop* treetop)
          return !comp()->getOption(TR_DisableMaxMinOptimization);
       case TR::java_lang_StringUTF16_toBytes:
          return !comp()->compileRelocatableCode();
+      case TR::java_lang_StrictMath_sqrt:
+      case TR::java_lang_Math_sqrt:
+         return true;
```

Comment:
Should we be checking here for `TR::Compiler->target.cpu.getSupportsHardwareSQRT()`? i.e.

```
return TR::Compiler->target.cpu.getSupportsHardwareSQRT();
```